timings/timings.go

```diff
@@ -80,6 +80,14 @@ func New(tags map[string]string) *Timings {
 	}
 }
 
+// AddTag sets a tag on the Timings object.
+func (t *Timings) AddTag(tag, value string) {
+	if t.tags == nil {
```

Comment:
Silly question, does this need synchronisation?

Reply:
No, this is always used on an instance of Timings object created locally for given go routine (e.g. at the beginning of task handler, or in ensure loop), this is the assumption of entire timings API.